bigtable/client/table.h

```diff
@@ -52,6 +52,7 @@ class Table {
    * The policies are passed by value, because this makes it easy for
    * applications to create them.  For example:
    *
+   * **Example**
```

Comment:
Please do **not** merge changes into your PR.  Use `git rebase`, as described in the CONTRIBUTING.md file.